cognite/neat/rules/exporter/rules2pydantic_models.py

```diff
@@ -495,21 +495,47 @@ def is_external_id_valid(external_id: str) -> bool:
 
     for edge_one_to_many in self.edges_one_to_many:
         edge_type_id = f"{self.__class__.__name__}.{edge_one_to_many}"
-
-        edges.extend(
-            EdgeApply(
-                space=data_model.space,
-                external_id=f"{self.external_id}-{end_node_id}",
-                type=(data_model.space, edge_type_id),
-                start_node=(data_model.space, self.external_id),
-                end_node=(
-                    data_model.space,
-                    end_node_id,
-                ),
+        if add_class_prefix:
+            edges_list = []
+            dm_view = data_model.views[self.__class__.__name__]
```

Comment:
Just my personal preference 
```suggestion
            dm_view = data_model.views[type(self).__name__]
```

Reply:
Ok, self.__class__.__name__ is used in the entire file, so I just kept the same style. I might change the entire file then, to keep it consistent.